core/src/main/res/values/strings.xml

```diff
@@ -709,4 +710,16 @@
     <string name="cast_failed_seek">Failed to seek to the new position on the cast device</string>
     <string name="cast_failed_receiver_player_error">Receiver player has encountered a severe error</string>
     <string name="cast_failed_media_error_skipping">Error playing media. Skipping&#8230;</string>
+
+    <!-- Notification channels -->
+    <string name="notification_channel_user_action">Action required</string>
+    <string name="notification_channel_user_action_description">Shown if your action is required</string>
+    <string name="notification_channel_downloading">Downloading</string>
+    <string name="notification_channel_downloading_description">Shown while currently downloading</string>
+    <string name="notification_channel_playing">Currently playing</string>
+    <string name="notification_channel_playing_description">Allows to control playback</string>
+    <string name="notification_channel_error">Errors</string>
+    <string name="notification_channel_error_description">Shown if something went wrong</string>
+    <string name="notification_channel_gpodnet">gpodder.net</string>
```

Comment:
Should probably be a capital letter. Adding examples to the descriptions would also be nice because they are displayed on a different screen anyways (not the preference summary)